hipe4ml/tree_handler.py

```diff
@@ -0,0 +1,275 @@
+"""
+Simple module with a class to manage the data used in the analysis
+"""
+
+import numpy as np
+import pandas as pd
+import uproot
+
+
+class TreeHandler:
+    """
+    Class for storing and managing the data of a ROOT tree from a .root file
+    or a pandas.DataFrame from a .parquet file
+    """
+
+    def __init__(self, file_name, tree_name=None, columns_names=None, **kwds):
```

Comment:
For both parquet and ROOT having the default columns_names =None might not be optimal. I would rather have the wildcard that reads all the columns in both cases.

Reply:
Hi @mpuccio, the method already implements that but i forgot to specify It. Thanks!